helm-charts/seldon-core-analytics/values.yaml

```diff
@@ -1,23 +1,42 @@
+grafana:
+  adminUser: admin
+  adminPassword: password
+  sidecar:
+    dashboards:
+      enabled: true
+      label: seldon_dashboard
+  datasources:
+    datasources.yaml:
+      apiVersion: 1
+      datasources:
+      - name: prometheus
+        type: prometheus
+        url: http://seldon-core-analytics-prometheus-seldon
```

Comment:
So now the prometheus service URL is http://seldon-core-analytics-prometheus-seldon or just http://prometheus-seldon as before?

Reply:
http://seldon-core-analytics-prometheus-seldon/ is now the service url. I made this change in order to know what the url was going to be ahead of time I had to follow the naming convention from the `helm install...` command. Is that an issue? Do other applications that use the url?

Reply:
Yes but I think it is better to use that convention and change everything else as needed.